src/Microsoft.DocAsCode.Build.SchemaDriven/Processors/MarkdownInterpreter.cs

```diff
@@ -38,7 +39,21 @@ private static string MarkupCore(string content, IProcessContext context, string
             (context.FileLinkSources).Merge(mr.FileLinkSources);
             (context.UidLinkSources).Merge(mr.UidLinkSources);
             (context.Dependency).UnionWith(mr.Dependency);
-            return mr.Html;
+
+            HtmlDocument htmlDocument = new HtmlDocument();
+            htmlDocument.LoadHtml(mr.Html);
+            foreach (var node in htmlDocument.DocumentNode.Descendants())
```

Comment:
No need to loop all the descendant nodes. Add it only the out most `<p>`.

Reply:
I verified and I think it's necessary because in <p> there may be several decents should be added new attribute. eg.:
<p sourcefile=\"inpihvwg.pun/landingPage1.yml\" sourcestartlinenumber=\"1\" sourceendlinenumber=\"1\" jsonpath=\"/sections/1/children/0/content\">Create an application using <a href=\"app-service-web-tutorial-dotnet-sqldatabase.md\" data-raw-source=\"[.NET with Azure SQL DB](app-service-web-tutorial-dotnet-sqldatabase.md)\" sourcefile=\"inpihvwg.pun/landingPage1.yml\" sourcestartlinenumber=\"1\" sourceendlinenumber=\"1\">.NET with Azure SQL DB</a> or <a href=\"app-service-web-tutorial-nodejs-mongodb-app.md\" data-raw-source=\"[Node.js with MongoDB](app-service-web-tutorial-nodejs-mongodb-app.md)\" sourcefile=\"inpihvwg.pun/landingPage1.yml\" sourcestartlinenumber=\"1\" sourceendlinenumber=\"1\">Node.js with MongoDB</a></p>